electronic_mail.py

```diff
@@ -46,6 +47,10 @@ def send_mail(cls, mails):
         Configuration = pool.get('electronic.mail.configuration')
         SMTP = pool.get('smtp.server')
 
+        if not PRODUCTION_ENV:
+            logger.warning('Production mode is not enabled.')
+            return
+
```

Comment:
I think the only thing that we should prevent is sending the e-mail. The rest of the code should behave normally.